### ISSUE TITLE
100 error when using in pipelines.

### ISSUE THREAD

**Issue by sunscreem** [AEnterprise/rsync-deploy#9]
I can't deploy sites using this repo at the moment as they are failing.

Looks like it might be a problem with debian:

```
 Ign:1 http://deb.debian.org/debian stretch InRelease
  Get:2 http://deb.debian.org/debian stretch-updates InRelease [93.6 kB]
  Ign:3 http://security.debian.org/debian-security stretch/updates InRelease
  Get:4 http://deb.debian.org/debian stretch Release [118 kB]
  Err:5 http://security.debian.org/debian-security stretch/updates Release
    404  Not Found [IP: 151.101.194.132 80]
  Get:6 http://deb.debian.org/debian stretch Release.gpg [3177 B]
  Get:7 http://deb.debian.org/debian stretch/main amd64 Packages [7080 kB]
  Reading package lists...
  E: The repository 'http://security.debian.org/debian-security stretch/updates Release' does not have a Release file.
  The command '/bin/sh -c apt update' returned a non-zero code: 100
  ```

**Comment by tohn** [comment]
Run into the same issue. You can use [my fork instead](https://github.com/tohn/rsync-deploy). I'm using the latest alpine image instead of Debian.

**Comment by azusachino** [comment]
Well, switch to debian:stable-slim in the Dockerfile could solve this problem. I forked this repo too, [ref](https://github.com/azusachino/rsync-deploy) .

**Comment by magestacks** [comment]
@tohn @azusachino How about changing the existing configuration?

```yaml
      - name: Deploy to Server
        uses: AEnterprise/rsync-deploy@v1.0
        env:
          DEPLOY_KEY: ${{ secrets.AUTO_REFRESH_BLOG_BY_GITHUB_ACTION }}
          ARGS: -avz --delete --exclude='*.pyc'
          SERVER_PORT: '22'
          FOLDER: docs/.vuepress/dist
          SERVER_IP: ${{ secrets.IP }}
          USERNAME: root
          SERVER_DESTINATION: /home
```

**Comment by azusachino** [comment]
@magestacks I don't think so, it's not the problem with the configuration, simply because the base image debian:9.5-slim failed to run `apt update`; which might indicate that 9.5-slim is out-of-service.

**Comment by magestacks** [comment]
@azusachino Hello, because I am not so familiar with the actions process, can you tell me where I should modify to fix the process? 

Thanks.

**Comment by azusachino** [comment]
@magestacks is this clear enough...
1. clone the repo
2. modify the `Dockerfile`, switch to `debian:stable-slim`
3. commit and make a new release, say v1.0.2
4. modify your `deploy.yml`, switch to your repo and your version of `rsync-deploy`

**Comment by magestacks** [comment]
@azusachino Is your address pushed to the public warehouse? If so, can I use your repo directly.

**Comment by azusachino** [comment]
@magestacks sure, `uses: azusachino/rsync-deploy@v1.0.2`, the repo is [here](https://github.com/azusachino/rsync-deploy)

**Comment by magestacks** [comment]
@azusachino Thank you very much, I suggest you modify the project README to your own version, I believe more people need this as I do.

**Comment by magestacks** [comment]
> @magestacks sure, `uses: azusachino/rsync-deploy@v1.0.2`, the repo is [here](https://github.com/azusachino/rsync-deploy)

thanks, it works.

**Comment by tokigith** [comment]
Something that might be related in this comment on stack overflow

> In late April 2023, the Stretch repositories were removed from the main mirrors, so you’ll need to switch to archive.debian.org; see [apt-get update failed to fetch debian amd64 packages while building dockerfile from maven:3.5.2-jdk-8](https://unix.stackexchange.com/q/743839/86440) and [Security repo for Debian stretch not working anymore](https://unix.stackexchange.com/q/744401/86440) for details.

https://unix.stackexchange.com/a/371907

**Comment by AEnterprise** [comment]
this should be fixed in v1.0.2
